### ISSUE TITLE
Simple gamma

### ISSUE THREAD

**Issue by jcbyts** [HukLab/PLDAPS#12]
Using simple gamma mode breaks the PsychImaging pipeline

in `openScreen` line 238
` elseif isfield(p.trial.display.gamma, 'power')`
           ` PsychImaging('AddTask', 'FinalFormatting', 'DisplayColorCorrection', 'SimpleGamma');`

generates the following error on my windows and linux machines running matlab 2016b and PTB 3.0.13

`Error using PsychImaging (line 1436)`
`Call PsychImaging('PrepareConfiguration'); first to prepare the configuration phase!`

**Comment by jcbyts** [comment]
`PsychImaging('AddTask', 'FinalFormatting', 'DisplayColorCorrection', 'SimpleGamma');` should be called before opening the window.  I added code to check and do this before opening the PTB window

**Comment by jonaskn** [comment]
Yes, SimpleGamma wasn't really supported before (except for that bit a dead code). Your code would still break Datapixx. No gamma would be applied for the overlay colors in the simple gamma case. (~pds.datapixx.init line 114).
Please either implement that or at least throw an error in pds.datapixx.init

**Comment by jcbyts** [comment]
@jonaskn good catch. On it.

**Comment by jcbyts** [comment]
@jonaskn can you check my handiwork when you get a chance? 

lines 116-117 and 142-144 of pds.datapixx.init

Thanks

**Comment by jonaskn** [comment]
@jcbyts looks good to me. 
Thanks

**Comment by jonaskn** [comment]
@jcbyts 
I just commited  5f373e909e2983c4244a97683532d0ae6bffb495

I noticed that the codes now breaks when no gamma is provided due to line 
`109: isstruct(p.trial.display.gamma)`
and was also not backward compatible. Was not calling SetLookupTable when no gamma is defined anymore.

Also noticed that there was already code for simpleGamma in there, that assumed simpleGamma when p.trial.display.gamma is a number.
Given that trying to use that would have crashed before as well, we can assume that no one was using it, so I removed that option. I also added the option to add extended gamma parameters
